Python/PRP/Modifier/pyRandomSoundMod.cpp

```diff
@@ -0,0 +1,127 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pyRandomSoundMod.h"
+#include "pyRandomCommandMod.h"
+
+#include <PRP/Modifier/plRandomSoundMod.h>
+#include "pyModifier.h"
+#include "PRP/pyCreatable.h"
+
+
+/* plRandomSoundModGroup */
+PY_PLASMA_NEW(RandomSoundModGroup, plRandomSoundModGroup)
+
+PY_PLASMA_TYPE(RandomSoundModGroup, plRandomSoundModGroup, "plRandomSoundModGroup wrapper")
+
```

Comment:
Why no getters and setters for plRandomSoundModGroup?

Reply:
None exist, not even in Plasma: https://github.com/H-uru/Plasma/blob/master/Sources/Plasma/FeatureLib/pfAudio/plRandomSoundMod.h#L50-L65

Reply:
The fields are public in Plasma. We will need to be able to use these fields from Python in order to properly support kickable SFX.

Reply:
Well, gosh.